src/gias3/applications/__init__.py

```diff
@@ -17,5 +17,9 @@
     You should have received a copy of the GNU General Public License
     along with GIAS.  If not, see <http://www.gnu.org/licenses/>..
 """
+import os
+
+os.environ['ETS_TOOLKIT'] = 'qt4'
```

Comment:
I think you might want to check and see if it is set first so that you don't overwrite someones existing setting.

Reply:
Using the packages (`mayavi` and its dependencies) that this module currently relies on, I believe all other `ETS_TOOLKIT` options are going to cause the console scripts to crash. I've added a check, and a warning in the case that the user has already set this variable externally.